processing/src/main/java/io/druid/query/GroupByParallelQueryRunner.java

```diff
@@ -56,6 +56,8 @@
 
 public class GroupByParallelQueryRunner<T> implements QueryRunner<T>
```

Comment:
this class can do both single and parallel running, perhaps we should rename it to simply `GroupByQueryRunner`